exercises/sheet4/Alberto_Ander_Filip/Sheet4.tex

```diff
@@ -0,0 +1,944 @@
+	we will consider now $\pi^{-1}(\bm{r_0})$ as an affine
+	space in $\RR^p$ (i.e., considering $\pi\colon\RR^p\to\RR^q$.)	
+	Indeed, since $\pi$ is an affine function, 
+	it has an associated linear function $\overline{\pi}$,
+	and
+	$\dim(\pi^{-1}(\bm{r_0})) = \dim \ker(\overline\pi) =
+	 p - \dim \mbox{Im} \overline\pi$.
+	Since we are assuming that $P$ and $Q$ have full dimension,
+	${\dim\mbox{Im}\overline\pi=q}$.
+\end{proof}
+
+\begin{lemma} \label{lem:lemazo}
+ Let $v\in \RR^p$. Then the function $f_v: P\rightarrow \RR$ defined by $s\mapsto \max\limits_{s + tv\in P} \{ t \}$ is continuous. 
+\end{lemma}
+\begin{proof} 
+	Note that $f_v$ is indeed well defined: for any $s\in P$ the maximum $t$ does exist.%Don't make me prove this 
+	\par
+	Let $\bm{c^1}(x)\leq c^1_0,\dots, \bm{c^k}(x)\leq c^k_0$ be the facet-defining inequalities for $P$,
+	for $\bm{c^i}\in(\RR^p)^\star$.
+	Then $P$ is the region of points satisfying them all. \par
+	Without loss of generality we may assume that the $c^i$'s such that
+	$\bm{c^i}(v)>0$ are $\bm{c^1},\dots, \bm{c^j}$.
+	Given $s\in P, t\in \RR$ satisfying $s + tv \in P$, 
+	there exists an $\eps > 0$ such that 
+	$s + (t+ \eps)v\in P$ 
+	if and only if 
+	$\bm{c^i}(s+tv) < c^i_0$ 
+	for $i\in \{1,\dots, j\}$.\par %this is a simple computation
+	Then $\max$ in the definition of $f_v$ can be computed with
+	only $j$ inequalities:
+	\[ 
+	\max \{t\in \RR \, : \, s + tv\in P\} =
+	\max \left\{ t \in \RR \, : \, \bm{c^i}(s+tv) \leq c^i_0 \ \ \forall i \in \{1,\dots, j\}\right\} 
+	\]
+	Since we are computing a maximum restricted to inequalities,
+	it can be computed as
+	\[
+	f_v(s) = \min_{1\leq i \leq j} 
+	\Big\{\max \{ t \in \RR \, : \, \bm{c^i}(s+tv)\leq c^i_0\}\Big\}.
+	\]
+	For all $i\in\{1,\dots,j\}$
+	it is satisfied that 
+	$\max \{ t \in \RR \, : \, \bm{c^i}(s+tv)\leq c^i_0\} =
+	\frac{c^i_0-\bm{c^i}(s)}{\bm{c^i}(v)}$ for $1\leq i \leq j$. 
+	In consequence 
+	$f_v(s)=\min_{1\leq i \leq j} \frac{c^i_0-\bm{c^i}(s)}{\bm{c^i}(v)}$.
+	Then $f_v$ is continuous since it is the minimum of
+	finitely many continuous functions.
+\end{proof}
+
+We have the following corollary.
+
+\begin{corollary}
+	\label{lem:lemilla}
+	Given $s\in  {P}$ and $v\in\RR^p$ such that $s+v\in   P$.
+	There exists $U\subseteq   P$, 
+	an open set (with the topology of $  P$), $s\in U$,
+	and $\eps > 0$ such that $U+\eps v \subseteq   P$.
+\end{corollary}
+
+\begin{lemma} \label{lemm:dimm}
+    $\dim \Sigma(P,Q) \geq \dim P - \dim Q$
+\end{lemma} 
+\begin{proof}
+	Notice that the interior of P is non-empty, 
+	as $\dim P=p$.	
+	Let $u\in int \, P$ and $r=\pi(u)$. 
+	Then the fiber $\hat{P}\defs \pi^{-1}(r)$ 
+	is a polytope of dimension $p-q$. 
+	Indeed, 
+	$\hat{P}= P \cap (u + \ker \overline\pi)
+	\supset B \cap (u + \ker \overline\pi)$ 
+	for some open ball
+	$B\subset P$ centered in $u$. 
+	Trivially 
+	$\dim (B \cap (u + \ker \overline\pi)) = 
+	\dim \ker \overline\pi=p-q$.\par
+	Now take $\gamma$ a section. 
```

Comment:
Now let $\gamma:Q\to P$ be a section. (Always provide context to your symbols, even though it's redundant; you want to make reading your text as easy as possible.)